api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -169,13 +173,21 @@ public OfflineUserController(
     int completionChangeCount = 0;
     int accessLevelChangeCount = 0;
 
+    Map<String, Boolean> twoFactorAuthStatuses = directoryService.getAllTwoFactorAuthStatuses();
     for (DbUser user : userService.getAllUsersExcludingDisabled()) {
       userCount++;
+      if (!twoFactorAuthStatuses.containsKey(user.getUsername())) {
```

Comment:
FYI: as a minor optimization, I moved this up ahead of the access tier lookup. In practice, this didn't really affect the performance. Since this is minor, I'm assuming this change is fine without reapproval so I won't wait for re-review (but feel free to, if you happen to see it in time).

Reply:
OK - I don't see any problem here